docs/data/base/components/number-field/number-field.md

```diff
@@ -149,19 +148,19 @@ The `largeStep` and `smallStep` props can be specified to change the step when a
 </NumberField.Root>
 ```
 
-### Format
+## Format
 
 The `format` prop accepts [`Intl.NumberFormat` options](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat) to customize the formatting of the input value:
 
 {{"demo": "UnstyledNumberFieldFormat.js"}}
 
-### Scrubbing
+## Scrubbing
 
-The `ScrubArea` subcomponent lets users scrub the value with their pointer as a faster alternative to the stepper buttons. This is useful in high-density UIs, such as an image editor that changes the width, height, or location of a layer:
+The `ScrubArea` subcomponent lets users scrub the value with their pointer as a faster alternative to the stepper buttons. This is useful in high-density UIs, such as an image editor that changes the width, height, or location of a layer. You could wrap an icon or a `<label/>` in the `ScrubArea` component.
```

Comment:
Reading this I did not understand what "scrubbing" meant
Maybe we can be more explicit.